src/ai/backend/common/utils.py

```diff
@@ -325,3 +328,57 @@ def _hydrate_entry(self, line):
         if entry:
             return await self.remove_entry(entry)
         return False
+
+
+current_loop: Callable[[], asyncio.AbstractEventLoop]
+if hasattr(asyncio, 'get_running_loop'):
+    current_loop = asyncio.get_running_loop  # type: ignore
+else:
+    current_loop = asyncio.get_event_loop    # type: ignore
+
+
+class AsyncJanusWriter:
```

Comment:
Good refactoring!
Just a single fix please: Let's rename it `AsyncFileWriter`, since use of `janus` is an internal implementation details of this class.

Reply:
Also, could you add some test cases for this class using temporary files?

Reply:
Sure, thanks for comments, addressed them in the amendments to that commit, please review. Besides, I saw your suggestion about implementing a separate context manager for one-shot writes into file: I think it would be more readable to keep it as with blocks wrapped into functions, since there are several different ways used for writing into file and it would force the new context manager to have a complex and not obvious signature.